contexts/citizenship-v2.jsonld

```diff
@@ -38,42 +35,169 @@
         "birthDate": {"@id": "https://schema.org/birthDate", "@type": "http://www.w3.org/2001/XMLSchema#dateTime"},
-        "commuterClassification": "https://w3id.org/citizenship#commuterClassification",
         "familyName": "https://schema.org/familyName",
-        "formerNationality": "https://w3id.org/citizenship#formerNationality",
         "gender": "https://schema.org/gender",
         "givenName": "https://schema.org/givenName",
         "height": "https://schema.org/height",
         "image": {"@id": "https://schema.org/image", "@type": "@id"},
         "maritalStatus": "https://w3id.org/citizenship#maritalStatus",
         "marriageCertificateNumber": "https://w3id.org/citizenship#marriageCertificateNumber",
-        "marriageLocation": {"@id": "https://w3id.org/citizenship#marriageLocation", "@type": "@id"},
+        "marriageLocation": {"@id": "https://w3id.org/citizenship#marriageLocation", "@type": "@id"}
+      }
+    },
+
+    "PermanentResident": {
+      "@id": "https://w3id.org/citizenship#PermanentResident",
+      "@context": {
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "commuterClassification": "https://w3id.org/citizenship#commuterClassification",
+        "formerNationality": "https://w3id.org/citizenship#formerNationality",
         "permanentResidentCard": {"@id": "https://w3id.org/citizenship#permanentResidentCard", "@type": "@id"},
         "residence": {"@id": "https://w3id.org/citizenship#residence", "@type": "@id"},
         "residentSince": {"@id": "https://w3id.org/citizenship#residentSince", "@type": "http://www.w3.org/2001/XMLSchema#dateTime"}
       }
     },
 
-    "Person": "https://schema.org/Person",
+    "PermanentResidentCard": {
+      "@id": "https://w3id.org/citizenship#PermanentResidentCard",
+      "@context": {
+        "@protected": true,
 
-    "PostalAddress": {
-      "@id": "https://schema.org/PostalAddress",
+        "id": "@id",
+        "type": "@type",
+
+        "filingLocation": {"@id": "https://w3id.org/citizenship#filingLocation", "@type": "@id"},
+        "identifier": "https://schema.org/identifier",
+        "lprCategory": "https://w3id.org/citizenship#lprCategory",
+        "lprNumber": "https://w3id.org/citizenship#lprNumber"
+      }
+    },
+
+    "PermanentResidentCardCredential": {
+      "@id": "https://w3id.org/citizenship#PermanentResidentCardCredential",
       "@context": {
         "@protected": true,
 
-        "addressCountry": "https://schema.org/addressCountry",
-        "addressLocality": "https://schema.org/addressLocality",
-        "addressRegion": "https://schema.org/addressRegion"
+        "description": "https://schema.org/description",
+        "name": "https://schema.org/name"
       }
     },
 
-    "QuantitativeValue": {
-      "@id": "https://schema.org/QuantitativeValue",
+    "EmploymentAuthorizationPerson": {
+      "@id": "https://w3id.org/citizenship#EmploymentAuthorizationPerson",
```

Comment:
This is weird... I get why it's being done... but... weird.

Reply:
Yes, it is weird.  I wasn't sure what else to use that fit the pattern here.  Suggestions welcome.